docs/pages/foundations/animation/guidelines.js

```diff
@@ -0,0 +1,215 @@
+
+          [Learn more about easing functions](https://www.smashingmagazine.com/2021/04/easing-functions-css-animations-transitions/#cubic-b%C3%A9zier-functions)
+          `}
+        >
+          <SandpackExample
+            code={easeExample}
+            name="Ease Example"
+            hideEditor
+            hideControls
+            previewHeight={800}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+      <MainSection
+        name="Accessibility"
+        description={`
+        Animations should never cause a user harm or distract from a necessary task. However, inaccessible animations have the potential to cause seizures, motion sickness, or anxiety in users, so we must ensure we are creating accessible animations every time.
+
+        Below are 3 key considerations for accessibility. Note: these do not apply for “essential” animations, where [essential is defined](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-pause.html#essentialdef) as any animation where “if removed, would fundamentally change the information or functionality of the content, _and_ information and functionality cannot be achieved in another way that would conform.”
+`}
+      >
+        <MainSection.Subsection
+          title="Looping animations"
+          description={`
+          Ensure that any animation that is moving, blinking, scrolling, or auto-playing and lasts longer than 5 seconds has an affordance for pausing and resuming the animation or an option to hide the animation.
+
+          _Looping animations should be avoided whenever possible, as they can become distracting very quickly._
+
+          Further reading: [WCAG 2.0 Pause, Stop Hide requirements](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-pause.html)
+          `}
+          columns={2}
+        />
+        <MainSection.Subsection
+          title="Flashing animations"
+          description={`
+          Ensure that there are no flashing animations present, where flashing is defined as anything that flashes _more than three times in a one second period_.
+
+          Further reading: [WCAG 2.0 Three Flashes requirements](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-three-times.html)
+          `}
+          columns={2}
+        />
+        <MainSection.Subsection
+          title="Reduced motion user setting"
+          description={`
+          Any non-essential animations should automatically be disabled when a user enables the reduced motion setting on their device. We have provided a [useReducedMotion hook](https://gestalt.pinterest.systems/web/utilities/usereducedmotion) to make this easy to implement.
```

Comment:
The hook is only for web, so we should indicate that.

Also: are there iOS/Android resources around reducing motion that we can link in addition to MDN?

Reply:
Added for iOS, can't seem to find any docs on Android...

Reply:
I can't either, even on Material. That seems…odd 🤔